### PR TITLE
Update us.m3u

### PR DESCRIPTION
**Pull request by Tribalize**
Update tvg-language, group-title, removed 1 dead dup

### CODE REVIEW

**Comment by Dum4G** [comment]
Are those links are truly duplicates or just different CDNs to choose from? @MapGuy11

File: channels/us.m3u [hunk 1]
```diff
@@ -5490,17 +5490,17 @@ https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
 https://a.jsrdn.com/broadcast/447912f76b/+0000/low/c.m3u8
 #EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ULF3iOE.png" group-title="Family",Positiv TV (1080p)
 https://api.new.livestream.com/accounts/27460990/events/8973036/live.m3u8
-#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="" tvg-logo="" group-title="",Positiv TV
+#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/ULF3iOE.png" group-title="Family",Positiv TV
```
Comment (MapGuy11) [inline channels/us.m3u:5493]: Duplicate Links

File: channels/us.m3u [hunk 2]
```diff
@@ -5490,17 +5490,17 @@ https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
...
 https://player-api.new.livestream.com/accounts/27460990/events/8973036/live.m3u8
-#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="",Positiv TV
+#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="Family",Positiv TV
 https://player-api.new.livestream.com/accounts/27460990/events/8973036/live.m3u8?Faith
```
Comment (MapGuy11) [inline channels/us.m3u:5496]: Remove

File: channels/us.m3u [hunk 3]
```diff
@@ -5490,17 +5490,17 @@ https://a.jsrdn.com/broadcast/447912f76b/+0000/c.m3u8
...
-#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="",Positiv TV
+#EXTINF:-1 tvg-id="PositivTV.us" tvg-name="Positiv TV" tvg-country="US" tvg-language="English" tvg-logo="https://github.com/geonsey/Free2ViewTV/blob/master/images/logos/Positiv_520x520.png?raw=true" group-title="Family",Positiv TV
```
Comment (MapGuy11) [inline channels/us.m3u:5497]: Remove

Comment (Tribalize) [reply channels/us.m3u:5497]: I can remove it. channel wasnt new. all are in the list now. So from now on if going through and testing you want me to remove working dup links that have the same url?

File: channels/us.m3u (hunk 1 above)
Comment (Tribalize) [reply channels/us.m3u:5493]: removed extra links

**Review by MapGuy11 — APPROVED** [review]

**Pull request merged**